### PR TITLE
add: `jonaburg-picom-git`

### PR DESCRIPTION
**Pull request by Ruturajn**
- Adding `jonaburg-picom-git` to the list of supported packages.
- `jonaburg-picom-git.pacscript` has been added.
- The packagelist is also updated with `jonaburg-picom-git`.

I had a question, since picom has a config file, where do I put it, since it needs to be stowed. Please suggest.

Thanks

### CODE REVIEW

**Comment by Henryws** [comment]
The config should not be included at all

**Comment by Ruturajn** [comment]
Ok, so should I remove those lines regarding placing the config file from the pacscript?

**Comment by Henryws** [comment]
> Ok, so should I remove those lines regarding placing the config file from the pacscript?

Yup

**Comment by Ruturajn** [comment]
Ok, I'll make the changes and push them.

File: packages/jonaburg-picom-git/jonaburg-picom-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Author : Ruturajn <nanotiruturaj@gmail.com>
+
```
Comment (Henryws) [inline packages/jonaburg-picom-git/jonaburg-picom-git.pacscript:2]: These lines aren't needed

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/jonaburg-picom-git/jonaburg-picom-git.pacscript (hunk 1 above)
Comment (Ruturajn) [reply packages/jonaburg-picom-git/jonaburg-picom-git.pacscript:2]: Ok, I'll remove them.

**Review by Henryws — APPROVED** [review]

**Comment by Ruturajn** [comment]
Hi, I wanted to add two more packages `ibhagwan-picom` and `qtile-extras` , so should I create a separate pull request for each one of them?

**Comment by Henryws** [comment]
Make separate PRs please!

**Comment by Ruturajn** [comment]
Ok, Thanks

**Pull request merged**